### PR TITLE
Purchase order bill

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 1]
```diff
@@ -5303,3 +5303,16 @@ form#groupDetailForm {
         color: inherit !important;
     }
 }
+
+
+.valid-tooltip{
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-1.scss:5308]: format css

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/gst/gst.component.scss [hunk 2]
```diff
@@ -405,15 +405,24 @@ hr {
 .header-info .btn-group > .dropdown-menu {
     margin-top: 0;
 }
+::ng-deep #monthPicker button.btn.btn-default.btn-info {
+    background: $chetwode-blue !important;
+    color: #fff!important;
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/gst/gst.component.scss:410]: Add color variable

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]
Resolve the conflicts

File: apps/web-giddh/src/app/gst/gst.component.scss (hunk 2 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/gst/gst.component.scss:410]: to be covered with code designing @shalinee-walkover

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/gst/gst.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gst.component.scss:410]: ok

**Review by shalinee-walkover — APPROVED** [review]
Approve

**Pull request merged**